src/routes/home/components/HomePage.js

```diff
@@ -30,3 +30,9 @@ internals.HomepageContainer = Styled.div`
     align-self: center;
     margin: auto;
 `;
+
+internals.WelcomeMessage = Styled(Typography).attrs({ variant: 'h4', align: 'center' })`
+
+    // Example leveraging the mui theme from inside a styled-component
+    color: ${(props) => props.theme.palette.duckYellow.main};
```

Comment:
It's purely stylistic, but this makes me personally feel warm/fuzzy.

```suggestion
    color: ${({ theme }) => theme.palette.duckYellow.main};
```

Reply:
Yeah, this is way better!